test/resources/application.conf

```diff
@@ -11,6 +11,7 @@ consignmentapi.url="http://localhost:9006/graphql"
 featureAccessBlock {
   closureMetadata=true
   descriptiveMetadata=true
+  viewTransfers=true
```

Comment:
This should be set to `false` as the page will show the button once the feature has been released, so this will reflect the final state, ie the feature will not be blocked following its release.

The final end start of the code will be to remove the feature blocks and the corresponding logic. This will occur once the feature has been successfully launched.

Reply:
As suggested, I have set the feature to false and updated the various tests